grype/db/v6/installation/curator.go

```diff
@@ -102,23 +103,15 @@ func (c curator) Status() db.Status {
 		}
 	}
 
-	var persistentErr error
-	digest, persistentErr := db.CalculateDBDigest(c.config.DBFilePath())
-
-	if validateErr := c.validate(); validateErr != nil {
-		if persistentErr != nil {
-			persistentErr = multierror.Append(persistentErr, validateErr)
-		} else {
-			persistentErr = validateErr
-		}
-	}
+	// override the checksum validation setting to ensure the checksum is always validated
+	digest, validateErr := c.validate(true)
```

Comment:
I think this makes it so `grype db status` always validates the checksum, so that it can report if the db has been edited since it was fetched. Is that right?

If so, minor nit, the comment should say something like "`grype db status` should always validate"; right now the comment doesn't tell me why we're not passing the config setting.

Reply:
righto -- I'll incorporate into another followup PR (this was automerged)